src/main/java/core/basesyntax/storage/Storage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.storage;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> dataBase = new HashMap<>();
```

Comment:
```suggestion
    public static final Map<Fruit, Integer> dataBase = new HashMap<>();
```
не критично, но лучше создать модель фрукта, в будущем может кроме name добавиться еще поле price или какое-то другое